libc-test/build.rs

```diff
@@ -286,6 +297,18 @@ fn main() {
             "QFMT_VFS_OLD" |
             "QFMT_VFS_V0" if mips && linux => true,
 
+            "USRQUOTA" |
+            "GRPQUOTA" |
+            "Q_SYNC" |
+            "Q_QUOTAON" |
+            "Q_QUOTAOFF" |
+            "Q_GETQUOTA" |
+            "Q_SETQUOTA" |
+            "RLIMIT_NPTS" |
+            "RLIMIT_SWAP" |
+            "RUSAGE_THREAD" |
+            "MADV_PROTECT"  if dragonfly => true,
```

Comment:
Ah ok, feel free to just delete any constants from the libc crate on dragonfly that aren't actually present in the dragonfly libc